packit_service/config.py

```diff
@@ -224,6 +224,23 @@ def get_project_to_sync(self, dg_repo_name, dg_branch) -> Optional[ProjectToSync
 
 
 class PackageConfigGetter:
+    @staticmethod
+    def create_issue_if_needed(
+        project: GitProject, message: str, details: str = ""
+    ) -> Optional[Issue]:
+        issues = project.get_issue_list(author=project.service.user.get_username())
+
+        if "[packit] Invalid config" in (x.title for x in issues):
```

Comment:
We don't have any filtering? (That would be handy.)

Reply:
only author/assignee

Reply:
thinking about it, `get_issue_list` fetches `open` issues by default, and if we filter it by `packit` user, it shouldn't be that big list 🤔